unicodetools/org/unicode/tools/emoji/CandidateData.java

```diff
@@ -825,7 +845,7 @@ public static void main(String[] args) {
 	DEBUG = true;
 	CandidateData candidateData = CandidateData.getInstance();
 	if (args.length == 0) {
-	    throw new IllegalArgumentException();
+	    args = new String[] {"proposals"};
```

Comment:
Modify the CandidateData to make it work properly whether we are in beta or not, and make the default to generate the proposals. Also added documentation above.